javascript/javascript2/week3/Game/game.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <title>Game</title>
+  <meta charset="UTF-8" />
+  <link rel="stylesheet" href="game.css">
+</head>
+
+<body>
+  <h1>Fastest presser</h1>
+  <input id="input" type="number" min="1" max="100">
```

Comment:
Try to guide the user by saying what to enter on this input, you can use `label` or `placeholder`

Reply:
Got it 👍🏻. Thank you for your feedback!